### PR TITLE
[#166239312] Set up Heroku deployment 

### PR DESCRIPTION
**Pull request by Emmanuel-Dominic**
#### What does this PR do?
 - Set up Heroku deployment 

#### Description of Task to be completed?
 - Currently, we do not have any deployment of the application. There is a need to deploy the current state of the application on Heroku so as to monitor progress and have visibility of the working system.

#### How should this be manually tested?
 - Simply click on this link to access the Heroku application URL [Heroku Link](https://wger-kronos.herokuapp.com)

#### What are the relevant pivotal tracker stories?
- [#166239312](https://www.pivotaltracker.com/n/projects/2350327/stories/166239312)

### CODE REVIEW

**Review by Rhytah — COMMENTED** [review]
@Emmanuel-Dominic could you enlighten me about the reason for the change from using the default `bower-components` static files to using  `cdnjs.cloudflare.com` static files

**Review by reifred — APPROVED** [review]
Thanks @Emmanuel-Dominic, for the work done. The deployed app is working fine.

**Comment by Manorlds-Eaglespark** [comment]
What could be the reason as to why the social share buttons are not displaying?
Screenshot 2019-06-05 at 13 00 47

**Review by Rhytah — APPROVED** [review]
@Emmanuel-Dominic It is safe to say that we have achieved successful Heroku deployment. Well done.

**Review by Manorlds-Eaglespark — APPROVED** [review]

**Comment by AGMETEOR** [comment]
@Emmanuel-Dominic I don't see your response Arnold's comment about social share buttons not displaying.

File: .gitignore [hunk 1]
```diff
@@ -46,7 +46,7 @@ coverage.xml
 
 # Django stuff:
 *.log
-settings*.py
+# settings*.py
```
Comment (AGMETEOR) [inline .gitignore]: @Emmanuel-Dominic why is commented out line still here?

File: settings.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    'NAME': os.environ.get('NAME'),
+    'USER': os.environ.get('USER'),
+    'PASSWORD': os.environ.get('PASSWORD'),
+    'HOST': os.environ.get('HOST'),
+    'PORT': '5432',
+    }
+}
+
+# Make this unique, and don't share it with anybody.
+SECRET_KEY = 'qi8da5)fd@k^3!mr5l85^)f+#$w%9jq2c&4rg)-s=w3)!dwnd4'
```
Comment (AGMETEOR) [inline settings.py]: What happened to not sharing this with anybody?

File: settings.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+# Your reCaptcha keys
+RECAPTCHA_PUBLIC_KEY = ''
+RECAPTCHA_PRIVATE_KEY = ''
+NOCAPTCHA = True
+
+# The site's URL (e.g. http://www.my-local-gym.com or http://localhost:8000)
+# This is needed for uploaded files and images (exercise images, etc.) to be
+# properly served.
+# SITE_URL = 'http://localhost:8000'
```
Comment (AGMETEOR) [inline settings.py]: Remove commented out code unless it's necessary

**Comment by AGMETEOR** [comment]
@Emmanuel-Dominic what is the relevance of the screenshot on the PR description?

**Comment by Emmanuel-Dominic** [comment]
> What could be the reason as to why the social share buttons are not displaying?
> Screenshot 2019-06-05 at 13 00 47

Yes @Manorlds-Eaglespark, Thanks for the feedback, i have implemented it successfully
Screenshot 2019-06-05 at 13 51 56

**Comment by Emmanuel-Dominic** [comment]
> @Emmanuel-Dominic I don't see your response Arnold's comment about social share buttons not displaying.

This was resolved with:
Screenshot 2019-06-05 at 13 51 56

File: .gitignore (hunk 1 above)
Comment (Emmanuel-Dominic) [reply .gitignore]: - Ensure resolved

File: settings.py (hunk 2 above)
Comment (Emmanuel-Dominic) [reply settings.py]: - Ensure resolved with the setting of the environmental variables

File: settings.py (hunk 3 above)
Comment (Emmanuel-Dominic) [reply settings.py]: - Resolved but i only left comments which redirect what is required or what the fields do.

**Comment by Emmanuel-Dominic** [comment]
> @Emmanuel-Dominic what is the relevance of the screenshot on the PR description?

- Thank you, this is resolved.

**Pull request merged**